api/cpp/esp-idf/slint/idf_component.yml

```diff
@@ -0,0 +1,7 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+description: "Slint — declarative GUI toolkit"
```

Comment:
Since this isn't bare-bones Slint but the ESP-IDF port, I think the description here and the README.md should emphasise that. It might still make sense to upload a pure slint component later.

Reply:
I don't see the point of separating Slint and its port as different idf components.